app/overrides/add_user_groups_to_users_menu.rb

```diff
@@ -0,0 +1,4 @@
+Deface::Override.new(:virtual_path  => "spree/admin/shared/_main_menu",
+                     :name          => "add_user_groups_to_users_menu",
+                     :replace       => "erb[loud]:contains('tab *Spree::BackendConfiguration::USER_TABS')",
+                     :erb          => "<%= main_menu_tree Spree.t(:users), icon: 'user', sub_menu: 'user', url: '#sidebar-users'")
```

Comment:
Line is too long. [130/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.